src/main/java/io/phasetwo/service/auth/idp/HomeIdpDiscoveryAuthenticatorFactory.java

```diff
@@ -16,81 +11,86 @@
 import org.keycloak.provider.ProviderConfigProperty;
 import org.keycloak.provider.ServerInfoAwareProviderFactory;
 
-@AutoService(AuthenticatorFactory.class)
-public final class HomeIdpDiscoveryAuthenticatorFactory
-    implements AuthenticatorFactory, ServerInfoAwareProviderFactory {
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
 
-  private static final Logger LOG = Logger.getLogger(HomeIdpDiscoveryAuthenticatorFactory.class);
+import static org.keycloak.models.AuthenticationExecutionModel.Requirement.*;
 
-  private static final AuthenticationExecutionModel.Requirement[] REQUIREMENT_CHOICES =
-      new AuthenticationExecutionModel.Requirement[] {REQUIRED, ALTERNATIVE, DISABLED};
+public final class HomeIdpDiscoveryAuthenticatorFactory implements AuthenticatorFactory, ServerInfoAwareProviderFactory {
```

Comment:
@phamann One thing I had to add to get it working was to add back in the `AutoService` annotation. I use this to generate the `META-INF/services` file. Can you add back? Here is the diff I made locally:
```
diff --git a/src/main/java/io/phasetwo/service/auth/idp/HomeIdpDiscoveryAuthenticatorFactory.java b/src/main/java/io/phasetwo/service/auth/idp/HomeIdpDiscoveryAuthenticatorFactory.java
index 0d6258b..263c976 100755
--- a/src/main/java/io/phasetwo/service/auth/idp/HomeIdpDiscoveryAuthenticatorFactory.java
+++ b/src/main/java/io/phasetwo/service/auth/idp/HomeIdpDiscoveryAuthenticatorFactory.java
@@ -1,6 +1,7 @@
 //package de.sventorben.keycloak.authentication.hidpd;
 package io.phasetwo.service.auth.idp;
 
+import com.google.auto.service.AutoService;
 import org.jboss.logging.Logger;
 import org.keycloak.Config;
 import org.keycloak.authentication.Authenticator;
@@ -17,6 +18,7 @@ import java.util.Map;
 
 import static org.keycloak.models.AuthenticationExecutionModel.Requirement.*;
 
+@AutoService(AuthenticatorFactory.class)
 public final class HomeIdpDiscoveryAuthenticatorFactory implements AuthenticatorFactory, ServerInfoAwareProviderFactory {
 
     private static final Logger LOG = Logger.getLogger(HomeIdpDiscoveryAuthenticatorFactory.class);
```

Reply:
Woops, good catch. Have fixed in: https://github.com/p2-inc/keycloak-orgs/pull/121/commits/b925e461e61e9e5113701ae04e66a240e777a884